internal/configs/parser_config_dir.go

```diff
@@ -88,7 +119,9 @@ func (p *Parser) loadFiles(paths []string, override bool) ([]*File, hcl.Diagnost
 	return files, diags
 }
 
-func (p *Parser) dirFiles(dir string) (primary, override []string, diags hcl.Diagnostics) {
+func (p *Parser) dirFiles(dir string, testsDir string) (primary, override, tests []string, diags hcl.Diagnostics) {
```

Comment:
It took me a while to understand what this function is doing now. While I think the previous function signature was basically self-explanatory, a function called `dirFiles` which takes two arguments isn't quite as intuitive. Perhaps a comment would help.

As a starting point, here's my understanding, which might of course be wrong:

> `dirFiles` finds Terraform configuration file paths which are direct descendants of `dir`, splitting them into primary files and overrides based on filename. If `testsDir` is non-empty and a sub-directory of that name exists, `dirFiles` also finds test file paths in that sub-directory and returns them separately.

Reply:
Added a comment!